source/features/hide-empty-split-diff-side.tsx

```diff
@@ -0,0 +1,31 @@
+import './hide-empty-split-diff-side.css';
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+function init(): void {
+	for (const diffTable of select.all('.js-diff-table')) {
+		for (const side of ['left', 'right']) {
+			if (!select.exists(`[data-split-side="${side}"]:is(.blob-code-addition, .blob-code-deletion)`, diffTable)) {
+				diffTable.classList.add('rgh-empty-split-' + side);
+				break;
+			}
+		}
+	}
+}
+
+void features.add(__filebasename, {
+	include: [
+		pageDetect.isCommit,
+		pageDetect.isCompare,
+		pageDetect.isPRFiles
+	],
+	exclude: [
+		() => !select.exists([
+			'[value="split"][checked]', // Form in PR
+			'.table-of-contents .selected[href*="diff=split"]' // Link in single commit
```

Comment:
What is this excluding its not clear from the comments

Reply:
The pages where there is no "Split diff" button selected. Taken from here:
https://github.com/sindresorhus/refined-github/blob/e948429b30fc204e98f1ba5389c561d86d109287/source/features/faster-pr-diff-options.tsx#L28-L29
(with `split` instead of `unified`)

Reply:
What do you this about adding a comment `The following pages there is no "Split diff" button.`?

Reply:
Rather, it's excluding pages that are _not_ in split diff. It's best to move this into a function at the top level:

```js
function isUnifiedDiff() {
	return exists(unified:checked, etc)
}
```